lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
>  as you suggested all commands need be changed to follow it;

Misunderstanding. My suggestion will work automatically for every new command. See second code snippet. Additional parameter is optional when command wants to build custom table.

>is really a good idea no make this explicit for each command in the 
Not sure I understand this correctly, but I think that this is not explicit. 3th parameter will be optional.